nipype/interfaces/afni/tests/test_extra_Deconvolve.py

```diff
@@ -7,4 +7,5 @@ def test_x1dstop():
     deconv.inputs.out_file = 'file.nii'
     assert 'out_file' in deconv._list_outputs()
     deconv.inputs.x1D_stop = True
-    assert not 'out_file' in deconv._list_outputs()
\ No newline at end of file
+    assert not 'out_file' in deconv._list_outputs()
+    assert not 'cbucket' in deconv._list_outputs()
```

Comment:
The syntax `x not in y` is preferred to `not x in y`. Could you change both of these lines?

Reply:
Patched and rebased into the previous commit!